addons/payment_stripe/static/src/js/payment_form.js

```diff
@@ -87,6 +87,9 @@ paymentForm.include({
         const paymentElement = this.stripeElements[paymentOptionId].create(
             'payment', paymentElementOptions
         );
+        paymentElement.on('loaderror', (response) => {
+            this._displayErrorDialog(_t("Incorrect payment details"), response.error.message);
```

Comment:
Same as for Adyen, and it makes more sense in this context.
```suggestion
            this._displayErrorDialog(_t("Cannot display the payment form"), response.error.message);
```